_data/muni_where_to_ride.yaml

```diff
@@ -1,89 +0,0 @@
-# vim: set ts=2 sw=2 softtabstop=2 expandtab:
```

Comment:
The commit message doesn't really match what you're doing here.

Reply:
I'm not sure how to change that retrospectively. I picked the _data directory as a test sort of arbitrarily because git was previously giving me issues. The main idea was that the content of the website was being refreshed, as noted in the subsequent commit message which completes the change.

Reply:
Test commits do not belong in deployed code. To retrospectively fix this you could use git interactive rebase to remove this commit from your history and then do a force push to your fork of this repo.

See https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History